dist/index.html

```diff
@@ -83,8 +95,8 @@ <h2 class="card-title">Tonic</h3>
                     </div>
                   </div>
                   
-                  <div class="grid-container card-2">
-                    <div class="card-container">
+             <div class="grid-container grid-container-desktop">
```

Comment:
- [ ] kindly note that while working in the desktop version, your mobile version should still look good like in the original design. please, kindly centralize your cards .
![Screenshot (1055)_LI](https://user-images.githubusercontent.com/65298048/236393682-00af9ad7-f2aa-4d92-9760-abaab69800c5.jpg)